starcheck/src/lib/Ska/Starcheck/Obsid.pm

```diff
@@ -2499,35 +2509,25 @@ sub time2date {
 }
 
 ###################################################################################
-sub count_good_stars{
+sub count_guide_stars{
 ###################################################################################
     my $self=shift;
     my $c;
-    my $clean_acq_count = 0;
-    my $clean_gui_count = 0;
-    $self->{count_nowarn_stars}{ACQ} = $clean_acq_count;
-    $self->{count_nowarn_stars}{GUI} = $clean_gui_count;
+    my $gui_count = 0;
 
     return unless ($c = find_command($self, 'MP_STARCAT'));
     for my $i (1 .. 16){
-        my $type = $c->{"TYPE$i"};
-        next if ($type eq 'NUL');
-        next if ($type eq 'FID');
-	if ($type =~ /ACQ|BOT/){
-	    unless ($self->check_idx_warn($i)){
-		$clean_acq_count++;
-	    }
-	}
-	if ($type =~ /GUI|BOT/){
-	    unless ($self->check_idx_warn($i)){
-		$clean_gui_count++;
-	    }
+	if ($c->{"TYPE$i"} =~ /GUI|BOT/){
+            my $mag = $c->{"MAG$i"};
+            # Use the fractional magnitudes from ORViewer.  tab-ternary is if/elsif/else
+            my $star_contrib = $mag <= 10.0  ? 1
```

Comment:
Do you know if the ORviewer code does `<=` or `<` here?

Reply:
Yes, I'd checked.   
```
            nPerAcq = sum(acqMags<starSettings.Acq.MinPerMag);
            nAcq = sum(acqMags<=starSettings.Acq.MinMagFull)...
                + .75*sum(acqMags>starSettings.Acq.MinMagFull & acqMags<=starSettings.Acq.MinMag75)...
                + .5*sum(acqMags>starSettings.Acq.MinMag75 & acqMags<=starSettings.Acq.MinMagHalf);
            
            nPerGui = sum(guiMags<starSettings.Gui.MinPerMag);
            nGui = sum(guiMags<=starSettings.Gui.MinMagFull)...
                + .75*sum(guiMags>starSettings.Gui.MinMagFull & guiMags<=starSettings.Gui.MinMag75)...
                + .5*sum(guiMags>starSettings.Gui.MinMag75 & guiMags<=starSettings.Gui.MinMagHalf);
```